sycl/include/CL/sycl/handler.hpp

```diff
@@ -286,6 +286,22 @@ class handler {
       case access::target::local: {
         detail::LocalAccessorBaseHost *LAcc =
             static_cast<detail::LocalAccessorBaseHost *>(Ptr);
+        // Stream implementation creates local accessor with size per work item
+        // in work group. Number of work items is not available during stream
+        // construction, that is why size of the accessor is updated here using
+        // information about number of work items in the work group.
+        if (detail::getSyclObjImpl(*LAcc)->PerWI &&
+            MNDRDesc.GlobalSize.size() != 1 &&
+            MNDRDesc.GlobalSize.size() != 1) {
```

Comment:
Maybe there is typo in one of the lines?

Reply:
You are right, fixed.